packages/image_picker/image_picker/README.md

```diff
@@ -74,7 +74,7 @@ Future<void> getLostData() async {
 }
 ```
 
-There's no way to detect when this happens, so calling this method at the right place is essential. We recommend to wire this into some kind of start up check. Please refer to the example app to see how we used it.
+There's no way to detect when this happens, so calling this method at the right place is essential. We recommend to wire this into some kind of start up check. Please refer to the [example](https://github.com/flutter/plugins/blob/main/packages/image_picker/image_picker/example/lib/main.dart) app to see how we used it.
```

Comment:
While I know what it's trying to say, I find the first part of this section really confusing; there is a way to detect it, because the code above does exactly that. How about replacing this whole paragraph with:

```
This check should always be run at startup in order to detect and handle
this case. Please refer to the
[example app](https://pub.dev/packages/image_picker/example)
for a more complete example of handling this flow.
```

(I think it's best to link to the pub view of the example, since that's where people are likely to be reading the README)

Reply:
Yes that is much better indeed, thanks.